tools/ctl/ctl.go

```diff
@@ -556,6 +556,9 @@ var runOne = &cobra.Command{
 		}
 		var strategy *schema.StrategyOneOf
 		if *strategyPath != "" {
+			if mode == firestore.AttestMode {
+				log.Fatal("Use --strategy-from-repo")
```

Comment:
Maybe mention that flag is required because of the attest mode?

"attest does not support providing a strategy, use --stragey-from-repo instead"?